parent/pom.xml

```diff
@@ -63,7 +63,8 @@
     <version.wildfly26>26.0.1.Final</version.wildfly26>
     <version.wildfly26.core>18.0.4.Final</version.wildfly26.core>
 
-    <version.tomcat>9.0.89</version.tomcat>
+    <version.tomcat9>9.0.89</version.tomcat9>
+    <version.tomcat>10.1.18</version.tomcat>
```

Comment:
❓I only noticed this with the backport: Can we update to latest available 10.1.x version? Right now it's 10.1.25.

Reply:
Absolutely, here is the [version bump](https://github.com/camunda/camunda-bpm-platform/pull/4342/commits/db997a3af08ffa02ed0704efe1e46a8414224f16).